### PR TITLE
Added webpack / browserify support

### PR DESCRIPTION
**Pull request by SuperPaintman**
Hi!
I fix exports from module.

```js
const angular = require('angular');
const localStorage = require('angular-local-storage');

angular.module('App', [
  localStorage // <- exports string with module name
])
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-local-storage",
-  "version": "0.2.4",
```
Comment (simison) [inline package.json]: This is probably something the package maintainer should do and should be left out of PRs.

**Comment by SuperPaintman** [comment]
Sorry. Now i fixed webpack exports, without change in src :)

File: src/angular-local-storage.js [hunk 2]
```diff
@@ -379,11 +386,11 @@ angular
...
 
-      if (value === null && isDefined(def)) {
-        value = def;
-      } else if (isObject(value) && isObject(def)) {
-        value = extend(value, def);
-      }
+          if (value === null && isDefined(def)) {
+            value = def;
+          } else if (isObject(value) && isObject(def)) {
+            value = extend(def, value);
```
Comment (ealves-pt) [inline src/angular-local-storage.js]: You have reverted a change from PR #286. Any reason for that?

File: .gitignore [hunk 3]
```diff
@@ -6,3 +6,4 @@ npm-debug.log
 *.swp
 .idea
 coverage
+*.map
```
Comment (ealves-pt) [inline .gitignore]: If you had this to the ignore we will be removing the functionality added with this PR #148

**Comment by SuperPaintman** [comment]
@ealves-pt done :)

**Comment by ealves-pt** [comment]
It seems ok for me. I will merge this PR.

Thanks :+1:

**Pull request merged**